src/schemas/ba-sak/begrunnelse/begrunnelse.tsx

```diff
@@ -243,8 +244,12 @@ const begrunnelse = {
       options: {
         list: vilkår,
       },
-      validation: rule => lagNasjonalFeltObligatoriskRegel(rule).warning('Vilkår ikke valgt'),
-      hidden: context => !erNasjonalBegrunnelse(context.document),
+      validation: rule => [
+        rule.required().warning('Vilkår ikke valgt'),
+        lagInstitusjonVilkårRegel(rule),
+      ],
+      hidden: context =>
+        !erNasjonalBegrunnelse(context.document) && !erInstitusjonsBegrunnelse(context.document),
```

Comment:
Hadde det vært mest ryddig å heller snu om på denne og si at den skal skjules hvis det er eøsBegrunnelse? Jeg mått bruke to sekunder på å tenke på hva dette betydde 😆

Reply:
Ser nå at dette ble endret i en senere commit 👍